modules/redisearch.ts

```diff
@@ -739,16 +746,12 @@ export type FTAggregateParameters = {
         nargs: string,
         properties: string[]
     },
+    apply?: FTExpression[],
```

Comment:
I think you meant to change this from apply to expression according to LOC 728.
Can you align 1 name?

Reply:
sorry, comment was wrong.

FYI both are redis 'apply' both use type 'FTExpression' - if you would like naming to changed to something like `pre-expression` just shout.